lib/mayaUsd/ufe/UsdTransform3dMatrixOp.cpp

```diff
@@ -40,6 +41,8 @@ namespace {
 using namespace MayaUsd;
 using namespace MayaUsd::ufe;
 
+void warnUnimplemented(const char* msg) { TF_WARN("Illegal call to unimplemented %s", msg); }
```

Comment:
@ppt-adsk I don't see warnUnimplemented is being used?

Reply:
Another good catch!  Refactored to UsdSetXformOpUndoableCommandBase.cpp, but forgot to remove the original.  Removed locally.